vm/loader_test.go

```diff
@@ -174,3 +174,29 @@ func TestProcessEvents(t *testing.T) {
 
 	}
 }
+
+var testProgFiles = []string{
+	"test.wrongext",
+	"test.mtail",
+	".test",
+}
+
+func TestLoadProg(t *testing.T) {
```

Comment:
I was trying to figure out how to check the contents of type `Loader` to validate that it's not loading the two non-conforming filenames.  But it doesn't look like the struct knows.